### DISCUSSION TITLE
main.js:3 Uncaught TypeError: Cannot read properties of null (reading 'value')     at fetchPokemon

### DISCUSSION THREAD

**Discussion by XavCilia** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#364] [Q&A]
Hola explorer bonito sábado, recurro a su sabiduría...la consola me arroja 2 errores uno en el JS en value.toLowerCase y otro en button onclick, prácticamente se queda estática, alguien me pudiera ayudar :(

![Captura de pantalla 2022-03-12 a la(s) 12 03 34](https://user-images.githubusercontent.com/99370308/158030303-fec8e803-a722-4163-9909-6b4735a2797c.png)
![Captura de pantalla 2022-03-12 a la(s) 12 03 54](https://user-images.githubusercontent.com/99370308/158030309-5d9a80ed-749a-43df-8dbf-0a946de5e7f4.png)
![Captura de pantalla 2022-03-12 a la(s) 12 07 16](https://user-images.githubusercontent.com/99370308/158030322-aa1558c9-b9c3-4b8d-9cab-ea01070fda7b.png)
![Captura de pantalla 2022-03-12 a la(s) 12 04 26](https://user-images.githubusercontent.com/99370308/158030329-642643d9-324c-4d9b-bfba-7248dcb03df0.png)

**Comment by doguedogue** [comment]
Pon bien tu ID en el input de pokeName veo dos comillas dobles

**Reply by XavCilia** [reply]
Cierto era eso 🙆🏽‍♂️ muchas gracias!

**Comment by sergiolopezloya** [comment]
Aún no está publicada la práctica pero a mi también me quemaba hacer el pokedex, jajajaja
